package.json

```
@@ -167,7 +167,7 @@
     "react-native-webview": "^11.3.1",
     "react-relay": "12.0.0",
     "react-spring": "8.0.23",
-    "react-tracking": "7.2.1",
+    "react-tracking": "^9.1.0",
```

```suggestion
    "react-tracking": "9.1.0",
```

we are really trying to conform with this idea of pinning exact versions, so i would suggest we keep the `^` out.

---

also, i have to say..


UPGRADES WOOP WOOP 💜 🥳 🎉

---

done